src/main/java/com/andersen/java_intensive_13/hotel_app/model/User.java

```diff
@@ -0,0 +1,34 @@
+package com.andersen.java_intensive_13.hotel_app.model;
+
+import com.andersen.java_intensive_13.hotel_app.enums.UserRole;
+import jakarta.persistence.*;
+import lombok.*;
+
+@Getter
+@Setter
+@ToString
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Builder
+@Table(name = "user", schema = "public")
+public class User extends AuditableEntity<Long> {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "first_name")
+    private String firstName;
+
+    @Column(name = "last_name")
+    private String lastName;
+
+    @Enumerated(EnumType.STRING)
+    @Column(name = "user_role")
+    private UserRole userRole;
+
+    public User(String fistName) {
+        this.firstName = fistName;
+    }
```

Comment:
you can use just builder instead of new constructor

Reply:
I delete it, it was extra constructor, not using in application.